packages/apollo-datasource-rest/src/connectors/memcached.ts

```diff
@@ -27,8 +29,17 @@ export default class MemcachedKeyValueCache implements KeyValueCache {
   ): Promise<void> {
     const { ttl, tags } = Object.assign({}, this.defaultSetOptions, options);
 
+    // get and incr version number
+    let version = await this.client.get(VERSION_KEY);
+    if (!version) {
+      // initialize version number
+      version = 1;
+      await this.client.set(VERSION_KEY, version + 1, 0);
+    } else {
+      await this.client.incr(VERSION_KEY, 1);
```

Comment:
I'm a little confused by the need to increment here, I would expect that to happen only on invalidation. What is the reasoning behind this?